blocks/hero/hero.js

```diff
@@ -9,27 +9,46 @@ export default function decorate(block) {
   const content = wrapper.querySelector('div');
   content.classList.add('hero-content');
   contentContainer.append(content);
-  const picture = content.querySelector('picture');
-  picture.parentElement.remove();
-  picture.classList.add('hero-background');
-  let tempPic;
-
-  if (picture.querySelector('img')) {
-    tempPic = createOptimizedPicture(
-      picture.querySelector('img')?.src,
-      'hero image',
-      true,
-      [
-        { media: '(min-width: 1600px)', width: 3000 },
-        { media: '(min-width: 600px)', width: 2500 },
-        { width: 750 },
-      ],
-    );
-
-    tempPic.classList.add('hero-background');
-  }
+  const pictures = content.querySelectorAll('picture');
+
+  pictures.forEach((picture, index) => {
+    picture.parentElement.remove();
+    picture.classList.add('hero-background');
+    let tempPic;
+
+    if (picture.querySelector('img')) {
+      tempPic = createOptimizedPicture(
```

Comment:
Don't use createOptimizedPicture - we have an optimzied picture. Just use it and if needed clone the DOM.

Reply:
The only time you should need to use `createOptimizedPicture` is if you have a URL reference to an image that you have to fetch - if the image reference is coming from the server as part of the block/page markup, it is already optimized, and has more data (i.e. dimensions) than this function can provide.